india_compliance/gst_india/utils/gstr_1/gstr_1_data.py

```diff
@@ -171,16 +173,30 @@ def get_query_with_common_filters(self, query):
         return query
 
     def calculate_totals(self, query, si_doc, key):
-        # TODO: Handle Refunds and TDS
+        # TODO: Handle TDS
+        refund_amount = (
+            frappe.qb.from_(self.si_taxes)
+            .select(
```

Comment:
https://stackoverflow.com/questions/2577174/join-vs-sub-query

Let's not use subq